pandas/io/sas/sasreader.py

```diff
@@ -107,6 +121,7 @@ def read_sas(
         .. versionchanged:: 1.2
 
             ``TextFileReader`` is a context manager.
+    %(decompression_options)s
```

Comment:
No indent here for the `%(...)`

Reply:
See question in other comment thread. I think your suggestion is wrong.